### PR TITLE
Addition of "Convert to Structured Grid" to gdas

### PR DESCRIPTION
**Pull request by jrocha22**
Added ConverttoStructuredGrid to gdas.cc

### CODE REVIEW

File: mains/gdas.cc [hunk 1]
```diff
@@ -52,6 +53,9 @@ int runApp(int argc, char** argv, const std::string traits, const std::string ap
   // Define a map from app names to lambda functions that create unique_ptr to Applications
   std::map<std::string, std::function<std::unique_ptr<oops::Application>()>> apps;
 
+  apps["convertstructuredgrid"] = []() {
```
Comment (CoryMartin-NOAA) [inline mains/gdas.cc]: may want to make this and the other entry `converttostructuredgrid` just to be consistent with the App name

File: mains/gdas.cc [hunk 2]
```diff
@@ -96,6 +100,7 @@ int main(int argc,  char ** argv) {
   // Check that the application is recognized
   // ----------------------------------------
   const std::set<std::string> validApps = {
+    "convertstructuredgrid",
```
Comment (CatherineThomas-NOAA) [inline mains/gdas.cc]: @jrocha22: I agree with @CoryMartin-NOAA's comment that this should probably be `converttostructuredgrid` for consistency.

File: mains/gdas.cc (hunk 1 above)
Comment (jrocha22) [reply mains/gdas.cc]: okay will do!

File: mains/gdas.cc (hunk 2 above)
Comment (jrocha22) [reply mains/gdas.cc]: sounds good!

Comment (CatherineThomas-NOAA) [reply mains/gdas.cc]: Looks great, thanks!

**Review by CatherineThomas-NOAA — APPROVED** [review]
Thanks for making the changes. Approve.

**Comment by CoryMartin-NOAA** [comment]
Kicking off the automated tests. If things look good enough, I'll merge!

**Review by danholdaway — APPROVED** [review]

**Pull request merged**